packages/pass-style/src/passStyleOf.js

```diff
@@ -205,9 +208,7 @@ const makePassStyleOf = passStyleHelpers => {
  * @type {PassStyleOf}
  */
 export const passStyleOf =
-  // UNTIL https://github.com/endojs/endo/issues/1514
-  // Prefer: globalThis?.VatData?.passStyleOf ||
-  (globalThis && globalThis.VatData && globalThis.VatData.passStyleOf) ||
+  (globalThis && globalThis[PassStyleOfEndowmentSymbol]) ||
   makePassStyleOf([
```

Comment:
It would be great if we could somehow avoid entraining all of passStyleOf's code into contract bundles, but that will likely require another approach in the future (I suppose something like an override of the package's import with a module that only looks up the global and fails otherwise)

Reply:
Not the intent of this PR at all. Willing to entertain at a later time, but seems low priority.

Reply:
Oh totally. Sorry that wasn't clear, it was more a random observation for the future.

Reply:
It's not to keep the endower's view simpler. It is to keep the normal client's view simpler.

Reply:
Thanks. Resolving.